exporter/otlp/lib/opentelemetry/exporter/otlp/exporter.rb

```diff
@@ -117,6 +118,11 @@ def send_bytes(bytes) # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticCompl
             # TODO: enable gzip when https://github.com/open-telemetry/opentelemetry-collector/issues/1344 is fixed.
             # request.add_field('Content-Encoding', 'gzip')
 
+            remaining_timeout = OpenTelemetry::SDK::Internal.maybe_timeout(timeout, start_time)
+            return FAILURE if remaining_timeout.zero?
```

Comment:
The spec only allows `SUCCESS` and `FAILURE` - there's no differentiation between `TIMEOUT` and `FAILURE`, unlike `force_flush` and `shutdown`.

Reply:
You've since added `TIMEOUT`, I think I should update this now right?